### ISSUE TITLE
`dao token new` does not allow token-name to have spaces in it

### ISSUE THREAD

**Issue by chrishobcroft** [aragon/aragon-cli#469]
## 🐛 Bug Report

I tried to run `dao token new` with a `token-name` _with spaces in it_, like so:

```
dao token new "Livepeer DAO Token" "LPDAO" 0 --environment aragon:rinkeby
```

It seems to have taken
`"Livepeer` as the `<token-name>`
`DAO` as the `<symbol>`
`Token` as the `[decimal units]`

### Have you read the [Contributing Guidelines on issues](https://github.com/aragon/aragon-cli/blob/master/CONTRIBUTING.md#ways-to-contribute)?

Yes

## Context

I am trying to deploy a DAO to Rinkeby using [@lkngtn's guide in the forum](https://forum.aragon.org/t/guide-custom-aragon-organization-deployment-using-the-cli/507).

#### Mainnet or testnet?

Rinkeby

#### Organization

Not applicable, as this is re: creating a new token.

#### Environment

- OS and OS version: Ubuntu 19.04
- Browser and browser version: N/A
- Add any other context about the problem here.

## To Reproduce

Run `dao token new "Livepeer DAO Token" "LPDAO" 0 --environment aragon:rinkeby`

## Expected behavior

I expected that the command would be able to create a new token with the name specified "inside the quote marks".

## Actual Behavior

The command returned with an error that the parameter for `decimals` must be an integer.

```
chrishobcroft@thinkpad:~$ dao token new "Livepeer DAO Token" "LPDAO" 0 --environment aragon:rinkeby
 ✔ Deploy the MiniMeTokenFactory contract
 ✖ Deploy the MiniMeToken contract
   → Error: [number-to-bn] while converting number "Token" to BN.js instance, error: invalid number value. Value must be an integer, hex string, BN or Bi…
 ✖ Error: [number-to-bn] while converting number "Token" to BN.js instance, error: invalid number value. Value must be an integer, hex string, BN or BigNumber instance. Note, decimals are not supported. Given value: "Token"
```

## Reproducible Demo

Run `dao token new "Livepeer DAO Token" "LPDAO" 0 --environment aragon:rinkeby`

**Comment by sohkai** [comment]
Haha yes... unfortunately this is a problem with parsing CLI arguments :(. Not sure if there's anything we can do for this, but related to https://github.com/aragon/aragon-cli/issues/369 cc @0x6431346e

**Comment by 0x6431346e** [comment]
Yeah that's the problem... if it's not too much trouble I'd recommend you to clone the aragonCLI and hardcode those params 😅

**Comment by xuhcc** [comment]
Bump.

**Comment by xuhcc** [comment]
The error is gone on `develop` branch :relieved:
